charts/hlf-k8s/templates/deployment-enrollement-operator.yaml

```diff
@@ -167,3 +168,4 @@ spec:
       nodeSelector:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+{{- end}}
```

Comment:
missing EOF, if you are using VSCode this option might help
```
"files.insertFinalNewline": true
```

Reply:
Fixed https://github.com/SubstraFoundation/hlf-k8s/pull/95/commits/45f3c3237f2ba2b6e5ad1d8769b5355547f5ca30

Thanks for the tip!